bin/configure_bind.php

```diff
@@ -21,11 +21,13 @@
     ]
 ];
 
-$context = stream_context_create($opts);
+if (!empty($rancherKey)) {
+    $opts['http']['header'] .= 'Authorization: Basic ' . base64_encode($rancherKey) . "\r\n";
+}
 
-$rancherKeyPrefix = $rancherKey ? $rancherKey . '@' : '';
+$context = stream_context_create($opts);
 
-$containers = json_decode(file_get_contents("http://{$rancherKeyPrefix}{$rancherHost}/{$version}/containers", false, $context));
+$containers = json_decode(file_get_contents("http://{$rancherHost}/{$version}/containers", false, $context));
 $self = json_decode(file_get_contents("http://{$selfHost}/{$versionDate}/self/container", false, $context));
```

Comment:
Are you sure that sending Authorization header to http://rancher-metadata works? I'm not able to test right now.